packages/angular-material/src/layouts/group-layout.renderer.ts

```diff
@@ -34,19 +34,34 @@ import { JsonFormsAngularService } from '@jsonforms/angular';
 @Component({
   selector: 'GroupLayoutRenderer',
   template: `
-    <mat-card appearance="outlined" fxLayout="column" [fxHide]="hidden">
+    <mat-card
+      [ngStyle]="{ display: hidden ? 'none' : '' }"
+      appearance="outlined"
+      class="group-layout"
+    >
       <mat-card-title class="mat-headline-6">{{ label }}</mat-card-title>
       <div
         *ngFor="
           let props of uischema | layoutChildrenRenderProps : schema : path;
           trackBy: trackElement
         "
-        fxFlex
       >
         <jsonforms-outlet [renderProps]="props"></jsonforms-outlet>
       </div>
     </mat-card>
   `,
+  styles: [
+    `
+      .group-layout {
+        display: flex;
+        flex-direction: column;
+        gap: 16px;
+      }
+      .group-layout > * {
```

Comment:
Should this really apply to all elements  or only `div`s? I ask because `mat-card-title` did not have `fxFlex` before

Reply:
I don't think it have an impact but to be on the safe side I'll specify div indeed.